util/src/test/java/io/camunda/zeebe/util/startup/StartupProcessTest.java

```diff
@@ -0,0 +1,382 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.util.startup;
+
+import static io.camunda.zeebe.util.sched.future.TestActorFuture.completedFuture;
+import static io.camunda.zeebe.util.sched.future.TestActorFuture.failedFuture;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.awaitility.Awaitility.await;
+import static org.mockito.Mockito.inOrder;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+import io.camunda.zeebe.util.sched.ConcurrencyControl;
+import io.camunda.zeebe.util.sched.TestConcurrencyControl;
+import io.camunda.zeebe.util.sched.future.ActorFuture;
+import io.camunda.zeebe.util.sched.future.TestActorFuture;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.CountDownLatch;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@SuppressWarnings({"rawtypes", "unchecked"})
+class StartupProcessTest {
```

Comment:
:wrench: I'd rather have the tests above the static classes/types of the steps, since when I look at this I want to focus on the tests and not the data classes.

Reply:
I agree. However, this is how Idea is sorting it for me.

Remember the discussion we had a couple of weeks ago in team meeting about which settings to set in Idea so that the code is uniformly formatted and sorted. Well, this is one of the instances where the default settings (to the best of my understanding) produce non-desirable outcomes)